scripts/migration/migrate_personal_to_profile_websites.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Script to migrate single value personal to profile_websites list."""
+
```

Comment:
I get an error when running this migration:

```
Traceback (most recent call last):
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/Users/sloria/miniconda/envs/osf/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/Users/sloria/projects/cos-projects/osf/scripts/migration/migrate_personal_to_profile_websites.py", line 38, in <module>
    main()
  File "/Users/sloria/projects/cos-projects/osf/scripts/migration/migrate_personal_to_profile_websites.py", line 26, in main
    logger.info("{}'s social dictionary is now {}".format(user.social, user.fullname))
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 19: ordinal not in range(128)
```

Users that have non-ascii characters in their name aren't being handled.

Either add the `u` prefix to that string, or do `from __future__ import unicode_literals` at the top of the file